mkt/api/paginator.py

```diff
@@ -39,13 +39,13 @@ def page(self, number):
         number = self.validate_number(number)
         bottom = (number - 1) * self.per_page
         top = bottom + self.per_page
-        page = Page(self.object_list[bottom:top], number, self)
 
         # Force the search to evaluate and then attach the count. We want to
         # avoid an extra useless query even if there are no results, so we
         # directly fetch the count from hits.
         # Overwrite `object_list` with the list of ES results.
```

Comment:
I wrote the original and find this change better and more straight forward. It's executing the ES query then creating the `Page` object from the results. Previously we created the `Page` object passing in the es-dsl `Search` object and then reaching inside and replacing `object_list` with the search results, which doesn't quite make sense.